platforms/android/xwalk.gradle

```diff
@@ -29,7 +29,18 @@ def DEFAULT_MIN_SDK_VERSION = 14
 
 def getConfigPreference(name) {
     name = name.toLowerCase()
-    def xml = file("res/xml/config.xml").getText()
+
+    def xml
+
+    // cordova-android >= 7.0.0
+    xml = file("src/main/res/xml/config.xml")
```

Comment:
What if you do something like

```
def configXmlLocations = [
    // cordova-android >= 7.0.0
    "src/main/res/xml/config.xml",
    // cordova-android < 7.0.0
    "res/xml/config.xml"
]
configXml = configXmlLocations.find {
    return file(it).exists()
}
xml = file(configXml).getText()
```

Reply:
It's less readable.

Reply:
I disagree. Right now you're using the same variable with different meanings, with calls to `file()` in different levels of indentation. You don't even need to follow what I said as-is (my Gradle knowledge is... modest), but the existing code could be improved.

Reply:
Pushed another one.